src/main/scala/uk/gov/hmrc/http/HttpPut.scala

```diff
@@ -25,11 +25,10 @@ import scala.concurrent.{ExecutionContext, Future}
 
 trait HttpPut extends CorePut with PutHttpTransport with HttpVerb with ConnectionTracing with HttpHooks with Retries {
 
-  override def PUT[I, O](
-    url: String,
-    body: I)(implicit wts: Writes[I], rds: HttpReads[O], hc: HeaderCarrier, ec: ExecutionContext): Future[O] =
+  override def PUT[I, O](url: String, body: I, headers: Map[String, String])
```

Comment:
For consistency it would be nice to keep the type of headers as `Seq[(String, String)]`?